rdrf/rdrf/views/registration_rdrf.py

```diff
@@ -144,6 +148,22 @@ def get_success_url(self, user=None):
 
 
 class PatientActivationView(ActivationView):
+
+    def custom_activation_key_expired(self, profile):
+        max_expiry_days = datetime.timedelta(days=settings.ACCOUNT_ACTIVATION_DAYS)
+        activation_date = profile.user.date_activated
+        expiration_date = (activation_date or profile.user.date_joined) + max_expiry_days
+
+        return profile.activated or expiration_date <= datetime.datetime.now()
+
+    def activate(self, *args, **kwargs):
+        activation_key = kwargs.get('activation_key', '')
+        user, activated = CustomRegistrationProfileAdmin.activate_user(self, activation_key)
```

Comment:
I'm not sure how I feel about having this method in the `CustomRegistrationProfileAdmin` class and calling it from here. Thoughts, @ScriptSmith ?